.circleci/config.yml

```diff
@@ -56,6 +64,103 @@ jobs:
 
 
 
+  build-linux-release:
+    <<: *defaults
+    docker:
+      - image: hyperledger/iroha-docker-develop
+    steps:
+      - checkout
+      - restore_cache:
+          keys:
+            - build-cache-release-{{ arch }}-{{ .Branch }}-
+            - build-cache-release-{{ arch }}-
+          paths:
+            - ~/.ccache
+      - run:
+          name: ccache setup
+          command: |
+            ccache --version
+            ccache --show-stats
+            ccache --zero-stats
+            ccache --max-size=1G
+      - run:
+          name: cmake
+          command: >
+            cmake \
+              -DCOVERAGE=OFF \
+              -DTESTING=OFF \
+              -H$IROHA_HOME \
+              -B$IROHA_BUILD \
+              -DCMAKE_BUILD_TYPE=Release \
+              -DPACKAGE_DEB=ON \
+              -DPACKAGE_TGZ=ON \
+              -DIROHA_VERSION=$(eval echo "$IROHA_VERSION")
+      - run:
+          name: make and package
+          command: |
+            cmake --build $IROHA_BUILD --target package -- -j4
+      - save_cache:
+          key: build-cache-release-{{ arch }}-{{ .Branch }}-{{ epoch }}
+          paths:
+            - ~/.ccache
+      - run:
+          name: copy deb file
+          command: |
+            IROHA_VERSION=$(eval echo "$IROHA_VERSION")
+            cd $IROHA_BUILD
+            # iroha-$IROHA_VERSION-Linux.{tar.gz, deb}, always matches 1 file
+            cp iroha-*.deb ./iroha.deb
+            cp iroha-*.tar.gz ./iroha.tar.gz
+            # if IROHA_VERSION is not set, then CIRCLE_SHA1
+            echo ${IROHA_VERSION:-$CIRCLE_SHA1} > version.txt
+            cd $IROHA_HOME
```

Comment:
Does this line affect next commands (`store_artifacts`, `persist_to_workspace`)? If not, change directory is not necessary here.

Reply:
It should not affect, because `run` spawns new terminal. It is safe to remove, I guess.